Chapter06/recipe-02/fortran-example/CMakeLists.txt

```diff
@@ -1,55 +1,37 @@
 cmake_minimum_required(VERSION 3.10 FATAL_ERROR)
 
-project(recipe-02 Fortran C)
+project(recipe-03 Fortran C)
```

Comment:
I think this one is still `recipe-02`. Otherwise all looks good! I'll fix up the recipe text on TC.

Reply:
Thanks for catching this.